release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -265,6 +265,11 @@ For more information, see xref:../networking/k8s_nmstate/k8s-nmstate-updating-no
 [id="ocp-4-14-machine-config-operator"]
 === Machine Config Operator
 
+[id="ocp-4-14-additional-prometheus-metrics"]
+==== Additional metrics available in Prometheus
+
+With this release, you can query additional metrics that let you more closely monitor the state of your machines and machine config pools.
+
```

Comment:
Since the  general term `additional metrics` is used, we could also provide a link to the documentation where these metrics are listed out.

Reply:
There isn't a doc that lists out the monitoring targets to that level of granularity, just a list of default targets. But there is a doc that details how to use Prometheus and view a list of available metrics, so maybe that could be linked here 🙂